### DISCUSSION TITLE
機車

### DISCUSSION THREAD

**Discussion by lai1008** [1101-GS4719-C/Python-Community-GS4719-C#2677] [General]
想問大家在中央有沒有自己的機車，這樣會不會方便很多，或是有什麼優缺點？

**Comment by Elaine4488** [comment]
我覺得有機車超方便的，尤其中央的公車班次不多，很難等。

**Comment by wcyeee** [comment]
我沒有自己的機車。但我覺得如果可以的話感覺有一台真的會蠻方便的！因為中央下山的公車不像台北那樣3分5分10分就來一班，我們的公車通常最久都要等30-40分鐘，如果有機車可以快速下山我是覺得會方便超級多，剩下排隊等車的時間，也可以避開公車上擠滿人的不適！

**Comment by 113607514** [comment]
我沒有自己的機車，但如果有一台，應該會方便許多！中央下山的公車不像台北那樣頻繁，每隔 3 分、5 分或 10 分就有一班。我們的公車通常需要等上 30 到 40 分鐘，真的很耗時間。如果有機車，不僅能快速下山，還能省下排隊等車的時間，也不用忍受公車上人擠人的不適，感覺生活會方便許多！

**Comment by forestdew507** [comment]
我沒有自己的機車。同意上面說如果有一台真的會更方便，因為公車不好等QQ 但就是中壢市區的交通要更注意安全 ( 光是坐公車都覺得有點可怕 )

**Comment by kzx-creat** [comment]
有機車超方便，去哪都行。然後我現在根本不會想搭公車，因為不用忍受班次少、公車甩尾、人擠人

**Comment by christaaah** [comment]
如果假日沒有要去哪裡，平常都待在宿舍並且對中央的食物沒意見或不介意叫外送的話，我覺得沒有機車也不會怎樣。如果感冒的時候可以搭公車去診所看醫生，遇到緊急情況的時候就搭Uber，但如果你想時常出去的話建議還是有機車比較方便。

**Comment by Mminniehuang** [comment]
有機車真的很方便，像是我平常有時候會工讀到晚上九點、十點，這個時候學校可以吃的選項真的沒那麼多，機車真的超級重要（對我來說😆）
缺點的話可能是因為平常都停在後門，每次停在棚子裡面都會是大晴天，不知道為什麼每次剛好停在外面就會下雨...

**Comment by yue083139** [comment]
我覺得有機車真的在中央是挺方便的，我原本也是搭公車去火車站，搭了幾次真的覺得好不喜歡，後來買了機車回家就變得很方便，不用在特定的時間趕公車可以隨時出門，要看病也能直接下山，出門的地點也不限於兩條公車線路，有時候可以下山吃點好的，像是上完python 3點學校就沒啥可吃的，沒事情要做就會想下山吃飯。

**Comment by Elaine4488** [comment]
擁有機車確實能大大提升在中央的便利性，特別是公車班次不多的情況下，機車讓你可以更自由地掌握時間，避免因為等車而浪費太多時間。此外，騎機車也讓你能更方便地前往附近的餐廳、超市或是其他學校周邊的地方，還可以隨時出門辦事，完全不受公共交通的時間限制。若是考慮到停車位和天氣等因素，這些不便其實也還算是可以接受的小挑戰。
